static/styles/popovers.css

```diff
@@ -302,7 +302,7 @@ ul {
         background-position: center;
         border-radius: 5px;
         margin-right: 10px;
-        border: 1px solid hsla(0, 0%, 0%, 0.2);
+        outline: 1px solid hsla(0, 0%, 0%, 0.2);
```

Comment:
@aadiljaleel switching from border to outline indeed solves the issue, so good work on spotting a simpler approach and not complicating it.

If I had to add anything, it would be a suggestion to use box-shadow with no blur `box-shadow: 0 0 0px 1px hsla(0, 0%, 0%, 0.2)` instead of the `outline` property

The reason for this suggestion is that we conventionally use the outline property as an accessibility feature, where we would want to highlight specific things to show that they have been selected. For this specific scenario, where we are using the border as a way to mimic shadow, the box-shadow property would be more conventionally accurate.

Reply:
Starting work on it

Reply:
Resolved in 4d29a2df